### PR TITLE
Bug 1417744, removed information about Pipelines being in tech preview

### PR DESCRIPTION
**Pull request by ahardin-rh**
https://bugzilla.redhat.com/show_bug.cgi?id=1417744

### CODE REVIEW

**Comment by ahardin-rh** [comment]
@adellape @thoraxe @jawnsy @jwforres PTAL

File: install_config/web_console_customization.adoc [hunk 1]
```diff
@@ -537,20 +537,6 @@ how to configure HAProxy routers to allow wildcard routes].
 Sometimes features are available in Technology Preview. By default, these
 features are disabled in the web console and hidden from end users.
 
-Web console features currently in Technology Preview include:
```
Comment (jwforres) [inline install_config/web_console_customization.adoc]: Should we say something like "There are currently no features in Technology Preview"

Comment (jwforres) [inline install_config/web_console_customization.adoc]: ... "that are hidden in the console", we may have things in tech preview that we don't need to hide in the console

Comment (ahardin-rh) [inline install_config/web_console_customization.adoc]: @jwforres Looks like there are 2 features for 3.4 that are listed in Technology Preview according to [3.4 release notes](https://docs.openshift.com/container-platform/3.4/release_notes/ocp_3_4_release_notes.html#ocp-34-kubernetes-deployments-support). Should those be listed here?

**Comment by jwforres** [comment]
Those aren't features that have a flag to show/hide them.  Pod disruption
budget doesn't show up at all in the console, Deployments will show up if
you add them.

**Comment by ahardin-rh** [comment]
@jwforres Thanks for clarifying. This is now updated.

**Comment by jwforres** [comment]
LGTM

**Comment by ahardin-rh** [comment]
Discussed the wording with @adellape via IRC yesterday. Merging now.

**Comment by ahardin-rh** [comment]
[rev_history]
|xref:../install_config/web_console_customization.adoc#install-config-web-console-customization[Customizing the Web Console]
|Removed information about Pipelines being a feature in Technology Preview.
%

**Pull request merged**